### PR TITLE
43 list incident action

### PR DESCRIPTION
**Pull request by daynewright**
## Status
**READY/IN DEVELOPMENT**

## Testing

[X] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Adds the menu for selecting incidents.  After selecting show incident view.

## Resolves Issue Number
#43 

```sh
git pull --prune
git checkout 43-list-incident-action
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.   Run
2.   Test various menu options.
3. 👀 Look at screen.

## Impacted Files in Application
List general components of the application that this PR will affect:

* ListIncidentAction.cs

### CODE REVIEW

**Review by matthamil — APPROVED** [review]
LGTM

File: src/customer-service/Actions/ListIncidentsAction.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            });
+            Console.Write("> ");
+            int selection = 0;
+
+            while (selection == 0)
+            {
+                try
+                {
+                    selection = Convert.ToInt32(Console.ReadLine());
+                    if (incidents.Find(i => i.IncidentId == selection).IncidentId == selection)
```
Comment (MattKraatz) [inline src/customer-service/Actions/ListIncidentsAction.cs:58]: this works. Could also have done maybe `incidents.Find(i => i.IncidentId == selection) != null`

File: src/customer-service/Actions/ListIncidentsAction.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    {
+                        IncidentFactory.Instance.ActiveIncident = incidentFact.get(selection);
+                        CustomerFactory.Instance.ActiveCustomer = custFact.get(orderFact.get(IncidentFactory.Instance.ActiveIncident.OrderId).CustomerId);
+                        ShowSingleIncidentAction.ReadInput();
+                        Console.Read();
```
Comment (MattKraatz) [inline src/customer-service/Actions/ListIncidentsAction.cs]: What does this do?

**Review by MattKraatz — APPROVED** [review]
LGTM

File: src/customer-service/Actions/ListIncidentsAction.cs (hunk 2 above)
Comment (daynewright) [reply src/customer-service/Actions/ListIncidentsAction.cs]: You Found the secret 🍳 !!

**Comment by garrett-vangilder** [comment]
Conflicts?

File: src/customer-service/Actions/ListIncidentsAction.cs (hunk 1 above)
Comment (daynewright) [reply src/customer-service/Actions/ListIncidentsAction.cs:58]: 👍

**Comment by garrett-vangilder** [comment]
:shipit:

**Comment by Anulfo** [comment]
LGTM

**Pull request merged**